sdk/core/azure_core/README.md

```diff
@@ -246,6 +246,101 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
 }
 ```
 
+### Consuming service methods returning `Poller<T>`
+
+If a service call may take a while to process, it would return `Result<Poller<T>>` as a result, representing a long-running operation (LRO).
```

Comment:
```suggestion
If a service call may take a while to process, it should return `Result<Poller<T>>` as a result, representing a long-running operation (LRO).
```
Alternate replacement word: `will` ("will" is stronger than "should")

Reply:
I actually waffled on "should" vs "would" and flat-out rejected "will" when I first wrote this little ditty for `Pager`. What concerns me is that the emitter might not always if the right LRO information isn't in TypeSpec, like I fear might be the case for Key Vault. It certainly *should*, but that felt more like a recommendation to crate developers compared to "would" that was meant to have a connotation like "will" to developers using our crates without the promise of actually returning a `Poller`. @ronniegeraghty or @RickWinter thoughts here? "will" just felt too much like a promise we can't really make.

Reply:
Not a hill I even want to climb so I'll just change it to "should".